spec/daru/io/importers/active_record_spec.rb

```diff
@@ -1,19 +1,24 @@
 RSpec.describe Daru::IO::Importers::ActiveRecord do
   include_context 'sqlite3 database setup'
   context 'without specifying field names' do
-    let(:order)  { %I[id name age] }
     let(:fields) { [] }
-    let(:data)   { [[1,2],%w[Homer Marge],[20, 30]] }
 
-    it_behaves_like 'sql activerecord importer'
+    it_behaves_like 'exact daru dataframe',
+      ncols: 3,
+      nrows: 2,
```

Comment:
Redis Importer very direly needs `ncols` and `nrows` checks. In order to maintain similaroity, `nrows` and `ncols` checks have been added to other modules too.